### ISSUE TITLE
Mac OS app results are deleted when save OSM

### ISSUE THREAD

**Issue by macumber** [NREL/OpenStudio#2399]

**Comment by macumber** [comment]
I'll see if I can reproduce on Windows, if not I'll kick this to @kbenne

**Comment by jmarrec** [comment]
I don't have this problem on my Mac. I've tested thoroughly.
Mine is installed to `/usr/local/openstudio-2.0.0` and I have write privileges:

```
$ ls -l
drwxr-xr-x   17 julien  staff    578 Jan  9 12:25 openstudio-2.0.0
```

**Comment by DavidGoldwasser** [comment]
Fixed in 2.0.3
